internal/discord/commands.go

```diff
@@ -98,7 +98,53 @@ func (d *Discord) Ban(s *discordgo.Session, i *discordgo.InteractionCreate) {
 //	user:		User
 //	reason:		string
 func (d *Discord) Unban(s *discordgo.Session, i *discordgo.InteractionCreate) {
-	return
+	optionMap := mapOptions(i)
```

Comment:
I think you can use [this](https://pkg.go.dev/github.com/bwmarrin/discordgo#Session.GuildBans) to access the ban list then use the argument's autocomplete field to suggest the ban list. Not entirely sure if this will work, but could be worth a try